docs/0.1/grid_on_splinter.md

```diff
@@ -116,33 +105,7 @@ steps 3 through 10.
    root@gridd-alpha:/# export GRID_SERVICE_ID=01234-ABCDE::gsAA
    ```
 
-4. Create a new organization, `myorg`.
-
-   ```
-   root@gridd-alpha:/# grid \
-   organization create 314156 myorg '123 main street' \
-    --metadata gs1_company_prefixes=314156
-   ```
-
-   This command creates and submits a transaction to create a new Pike
-   organization that is signed by the admin key. It also creates a new Pike
-   agent with the “admin” role for the new organization (this agent’s public key
-   is derived from the private key used to sign the transaction.) The service ID
-   includes the circuit name and the scabbard service name for the alpha node.
-
-5. Update the agent's permissions (Pike roles) to allow creating, updating, and
-   deleting Grid products.
-
-   ```
-   root@gridd-alpha:/# grid \
-   agent update 314156 $(cat ~/.grid/keys/alpha-agent.pub) --active \
-   --role can_create_product \
-   --role can_update_product \
-   --role can_delete_product \
-   --role admin
-   ```
-
-6. Use `cat` to create a product definition file, `product.yaml`, using the
+1. Use `cat` to create a product definition file, `product.yaml`, using the
    following contents.
 
    ```
```

Comment:
The format of this file changes in https://github.com/hyperledger/grid/pull/354. This PR also makes it necessary to create a schema for the product prior to adding the product. This will also make it necessary to add the role "can_create_schema" to the agent in the previous step.

Reply:
Thanks for this info, @dnewh -  I'll merge the current version now, then open a separate story to update it with the changes from  hyperledger/grid#354.